intro-HPC/ch_troubleshooting.tex

```diff
@@ -20,14 +20,77 @@ \section{Walltime issues}
 \section{Out of quota issues}
 
 Sometimes a job hangs at some point or it stops writing in the disk. These errors are usually
-related to the quota usage. You may have reached your quota limit at some storage endpoint. 
+related to the quota usage. You may have reached your quota limit at some storage endpoint.
 You should move (or remove) the data to a different storage endpoint (or request more quota) to be able to write to the disk and then resubmit the jobs.
 \ifgent
 Another option is to request extra quota for your VO to the VO moderator/s.
-See section~\ref{subsec:predefined-user-directories} and section~\ref{subsec:predfined-quotas} for more information about 
+See section~\ref{subsec:predefined-user-directories} and section~\ref{subsec:predfined-quotas} for more information about
 quotas and how to use the storage endpoints in an efficient way.
 \fi
 % FIXME: Add how to request quota section
 
+\section{Issues connecting to login node}
+\label{sec:connecting-issues}
 
+If you have issues connecting to the login nodes, here is a list of things to do
+that might help:
 
+\begin{enumerate}
+    \item Wait at least 1 hour after you VSC account has been \emph{approved}.
+\ifmacORlinux
+    \item Your ssh private key is not in the default location (\verb|~/.ssh/id_rsa|).
+        There are several ways to fix this:
+        \begin{enumerate}
+            \item Use the \verb|ssh -i| (see section~\ref{sec:connect})
+            \item Use \verb|ssh-add|:
+                \ifmac
+                see section~\ref{sec:using-ssh-agent-with-openssh}
+                \else
+                see this \href{https://apple.stackexchange.com/a/250572}{answer on StackExchange}.
+                \fi
+            \item Specify the location of the key in \verb|~/.ssh/config|:
+
+            \begin{prompt}
+Host %\loginnode%
+    IdentityFile %\emph{/path/to/private/key}%
+    User %\emph{your\_username}% # looks like %\userid%\end{prompt}
```

Comment:
use `\userid` straight away, and mention above that it should be replaced by "your own VSC login ID" (and refer to a section that explains how to determine your VSC login ID, maybe in the FAQ section?)

Reply:
Also use a shorthand like `hpcugent` and put full hostname below, like this:

```
Host hpcugent
  Hostname login.hpc.ugent.be
  ...
```

That gives the added benefit of being able to do `ssh hpcugent`.

That whole thing should probably in a separate subsection in chapter 3, where you can refer to here?